### DISCUSSION TITLE
v2.1.1

### DISCUSSION THREAD

**Discussion by ruslandoga** [plausible/analytics#4195] [Self-Hosted Support]
This is a patch release with fixes to some of the problems reported in [v2.1.0](https://github.com/plausible/analytics/discussions/4125)

### Upgrading to v2.1.1

> [!IMPORTANT]
> This guide assumes you are already using [v2.1.0](https://github.com/plausible/analytics/discussions/4125)

#### Update images

Update the image used for `plausible`

<sub><kbd>docker-compose.yml</kbd></sub>
```diff
plausible:
- image: ghcr.io/plausible/community-edition:v2.1.0
+ image: ghcr.io/plausible/community-edition:v2.1.1
```

and restart the container

<sub><kbd>console</kbd></sub>
```console
$ cd hosting
$ docker compose stop plausible
$ docker compose rm plausible
$ docker compose up -d
```

This will boot up the new version of the app.

## Changelog

Following [changes](https://github.com/plausible/analytics/compare/v2.1.0...v2.1.1) have been made since v2.1.0:

### Added
- Snippet integration verification
- Limited filtering support for imported data in the dashboard and via Stats API
- Automatic sites.imported_data -> site_imports data migration in CE https://github.com/plausible/analytics/pull/4155

### Fixed
- Fix CSV import by adding a newline to the INSERT statement https://github.com/plausible/analytics/pull/4172
- Fix url parameters escaping of = sign https://github.com/plausible/analytics/pull/4185
- Fix redirect after registration in CE https://github.com/plausible/analytics/pull/4165
- Fix VersionedSessions migration in ClickHouse v24 https://github.com/plausible/analytics/pull/4162

<hr /><em>This discussion was created from the release <a href='https://github.com/plausible/analytics/releases/tag/v2.1.1'>v2.1.1</a>.</em>

**Comment by oszfer** [comment]
Thanks a lot, it works well now.
